src/commands/test/cli_test.py

```diff
@@ -0,0 +1,20 @@
+import re
```

Comment:
`cli_test` might be not the best name for it. This file doesn't test the CLI but test runner, and we may want to test our CLI later on. Maybe `run_test_runner_test`?

Reply:
yeah good point - was pretty stumped with naming this one 😬